src/api-auth-form/submit-button.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var $ = require('jquery');
+
+var decoratorHelper = require('./decorator-helper');
+
+var DEFAULT_SUBMIT_BUTTON_CLASS = 'sso-submit';
```

Comment:
Default class should be btn-submit. This way we abstract the construction of buttons from this sepcific sso feature.